test/parallel/test-http-parser.js

```diff
@@ -129,7 +129,7 @@ function expectBody(expected) {
 //
 {
   const request = Buffer.from(
-      'HTTP/1.1 200 OK' + CRLF +
+    'HTTP/1.1 200 OK' + CRLF +
```

Comment:
That's the rule? just the first one, and then there's a continuation indent added...
Not amazing 🤷‍♂️